extensions/arc/src/ui/dialogs/connectControllerDialog.ts

```diff
@@ -149,3 +175,51 @@ export class ConnectToControllerDialog extends InitializingComponent {
 		return this._completionPromise.promise;
 	}
 }
+
+export class PasswordToControllerDialog extends ConnectToControllerDialog {
+
+	constructor(protected _treeDataProvider: AzureArcTreeDataProvider, title: string = loc.passwordToController) {
+		super(_treeDataProvider, title);
+	}
+
+	public async validate(): Promise<boolean> {
+		if (!this.passwordInputBox.value) {
+			return false;
+		}
+		const azdataApi = <azdataExt.IExtension>vscode.extensions.getExtension(azdataExt.extension.name)?.exports;
+		try {
+			await azdataApi.azdata.login(this.urlInputBox.value!, this.usernameInputBox.value!, this.passwordInputBox.value);
+		} catch (e) {
+			if (getErrorMessage(e).match(/Wrong username or password/i)) {
+				this.dialog.message = {
+					text: loc.invalidPassword,
+					level: azdata.window.MessageLevel.Error
+				};
+				return false;
+			} else {
+				this.dialog.message = {
```

Comment:
This message string mentions "taking corrective actions". But I don't understand what that would mean here - how is the user supposed to know what to do to "fix" it when the login fails? 

In general with my comment above I don't see why we're trying to do our own logic here instead of just displaying the error message from the login directly.

Reply:
Yeah I'd prefer to keep it simple - and "take corrective actions" is way too formal anyways. So just removing it seems easiest.